packages/@internationalized/date/src/queries.ts

```diff
@@ -180,13 +180,30 @@ export function getMinimumDayInMonth(date: AnyCalendarDate) {
   return 1;
 }
 
+const DAY_MAP = {
```

Comment:
Should this be exported to be used https://github.com/adobe/react-spectrum/pull/7363/files#diff-b598cd9f6e4569b26a010dc1b345368a20953c412f842b0c035f3cd661890792R24?

Reply:
I didn't think it was worth exporting since they're different packages and it would be public. Open to changing this if we think that's better though.